cmd/kubeadm/app/phases/bootstraptoken/node/token.go

```diff
@@ -46,15 +48,21 @@ func UpdateOrCreateTokens(client clientset.Interface, failIfExists bool, tokens
 		}
 
 		updatedOrNewSecret := bootstraptokenv1.BootstrapTokenToSecret(&token)
-		// Try to create or update the token with an exponential backoff
-		err = apiclient.TryRunCommand(func() error {
-			if err := apiclient.CreateOrUpdateSecret(client, updatedOrNewSecret); err != nil {
-				return errors.Wrapf(err, "failed to create or update bootstrap token with name %s", secretName)
-			}
-			return nil
-		}, 5)
+
+		var lastError error
+		err = wait.PollUntilContextTimeout(
+			context.Background(),
+			kubeadmconstants.APICallRetryInterval,
+			kubeadmconstants.APICallWithWriteTimeout,
```

Comment:
This timeout has also changed. Before(in backoff): about 155s -> Now: 40s
But this seems safe since it's just an API call.

Reply:
yeah, if the kubeadm client is waiting 3 minutes for a token API call, something is wrong.